### DISCUSSION TITLE
Add AmazonLinux 2023 DataSource

### DISCUSSION THREAD

**Discussion by childnode** [renovatebot/renovate#23334] [Suggest an Idea]
### Tell us more.

repology actually supports AL2 but will not support AL2023

for details about this problem of missing datasource in repology you might head over to
https://github.com/repology/repology-updater/pull/1320

so, my feature request is same same but different to https://github.com/renovatebot/renovate/issues/18339

[#new datasource](https://github.com/renovatebot/renovate/labels/new%20datasource)

**Comment by childnode** [comment]
may I poke @rarkins 🥹

**Reply by rarkins** [reply]
PRs welcome
